PsfLauncher/PsfPowershellScriptRunner.h

```diff
@@ -56,11 +66,39 @@ class PsfPowershellScriptRunner
 	//RunStartingScript should return an error only if stopOnScriptError is true
 	void RunStartingScript()
 	{
+		LogString("StartingScript commandString", this->startingScriptInformation.commandString.c_str());
+		LogString("StartingScript currentDirectory", this->startingScriptInformation.currentDirectory.c_str());
+		if (this->startingScriptInformation.runInVirtualEnvironment)
+		{
+			Log("StartingScript runInVirtualEnvironment=true");
+		}
+		else
+		{
+			Log("StartingScript runInVirtualEnvironment=false");
+		}
+		if (this->startingScriptInformation.waitForScriptToFinish)
+		{
+			Log("StartingScript waitForScriptToFinish=true");
+		}
+		else
+		{
+			Log("StartingScript waitForScriptToFinish=false");
+		}
 		RunScript(this->startingScriptInformation);
 	}
 
 	void RunEndingScript()
 	{
+		LogString("EndingScript commandString", this->endingScriptInformation.commandString.c_str());

```

Comment:
Same here with the runInVirtualEnvironment.

Reply:
Ditto.